ui/src/app/components/webcomponents/left-nav.component.html

```diff
@@ -4,68 +4,91 @@
     <a [routerLink]="['']">
     </a>
   </div>
-  <div class="text-center dropright mouse-over" (mouseover)="displayGlobalAdd=true" (mouseleave)="displayGlobalAdd=false">
-    <a class="primary-icon primary-icon-padding" href="javascript:void(0)">
+  <div class="text-center">
+    <div class="p-12"></div>
+    <div class="dropright mouse-over" (mouseover)="displayGlobalAdd=true" (mouseleave)="displayGlobalAdd=false">
+      <a class="primary-icon p-0" href="javascript:void(0)" style="height: unset">
```

Comment:
put empty href

Reply:
move styles to class names and try to avoid inline styles

Reply:
made changes